scripts/pre_commit_linter.py

```diff
@@ -78,6 +78,10 @@
     nargs='+',
     help='specific files to be linted. Space separated list',
     action='store')
+_PARSER.add_argument(
+    '-v',
+    help='v for verbose',
```

Comment:
Change this to `help='verbose mode. All details will be printed.'`

Reply:
Done

Reply:
Not done?

Reply:
Sorry for this.

Reply:
Done